hi-ml-cpath/src/SSL/encoders.py

```diff
@@ -91,4 +91,4 @@ def get_encoder_output_dim(
     with torch.no_grad():
         representations = pl_module(x)
 
-    return representations.shape[1]
+    return representations.shape[1]  # type: ignore
```

Comment:
Unrelated to this PR, but this is something I've been doing recently for TorchIO which I think is nicer than ignoring all errors: https://mypy.readthedocs.io/en/stable/error_codes.html#silencing-errors-based-on-error-codes